client/src/components/QA/AnswerItem.jsx

```diff
@@ -9,32 +11,81 @@ background: white;
 `;
 
 const Row = styled.div`
-    display: flex;
-    align-items: center;
+display: flex;
+align-items: center;
 `;
 
-const AnswerItem = ({ answer }) => (
-  <Wrapper>
-    <Row>
-      <h3>
-        A:&nbsp;
-      </h3>
-
-      <p>{answer.body}</p>
-    </Row>
-    <div>
-      <p style={{ margin: 0 }}>
-        by:&nbsp;
-        {answer.answerer_name}
-        ,&nbsp;
-        {moment(answer.date).format('LL')}
-        &nbsp;&nbsp;
-        |&nbsp;&nbsp;Helpful? Yes (
-        {answer.helpfulness}
-        )&nbsp;&nbsp;|&nbsp;&nbsp; Report
-      </p>
-    </div>
-  </Wrapper>
-);
+const Button = styled.button`
+background: none!important;
+border: none;
+padding: 0!important;
+font-weight: bold;
+text-decoration: underline;
+cursor: pointer;
+font-family: Valera Round,sans-serif;
+font-size: 16px
+`;
+
+const AnswerItem = ({ answer }) => {
+  const [helpful, setHelpful] = useState(answer.helpfulness);
+  const [upvoted, setUpvoted] = useState(false);
+  const [reported, setReported] = useState(false);
+
+  const handleUpvote = () => {
+    if (!upvoted) {
+      axios.put(`api/qa/answers/${answer.answer_id}/helpful`)
+        .then(() => {
+          setUpvoted(true);
+          setHelpful((prevState) => prevState + 1);
+        })
+        .catch((err) => {
+          throw err;
+        });
+    }
+  };
+
+  const handleReport = () => {
+    if (!reported) {
+      axios.put(`/api/qa/answers/${answer.answer_id}/report`)
+        .then(() => {
+          setReported(true);
+        })
+        .catch((err) => {
+          throw err;
+        });
+    }
```

Comment:
This looks very similar to my Helpful and Reported buttons. Maybe not the most ideal method, but still works!